salt/haproxy/haproxy.cfg.jinja

```diff
@@ -0,0 +1,28 @@
+global
+        log /dev/log    local0
+        log /dev/log    local1 notice
```

Comment:
They don't, really. haproxy cannot log to stdout. The "solutions" to this are not really solutions: https://github.com/dockerfile/haproxy/issues/3